hubspot/connection/__init__.py

```diff
@@ -13,16 +13,18 @@
 # INFRINGEMENT, AND FITNESS FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
+from __future__ import unicode_literals
```

Comment:
This line is the one that is causing problems with the tests. Could you please remove it?